charts/hono/requirements.yaml

```diff
@@ -19,3 +19,7 @@ dependencies:
     version: ~3.8.15
     repository: "https://kubernetes-charts.storage.googleapis.com/"
     condition: grafana.enabled
+  - name: mongodb
+    repository: "https://charts.bitnami.com/bitnami"
+    version: ~7.14.7
```

Comment:
which version of MongoDB does this use?

Reply:
It uses MongoDB version 4.2.8. It was the latest version of charts at the time of starting with helm charts for MongoDB device registry. Is the motive of the question to align with the version 4.0.20 that we use for the tests? I checked and there are no helm charts for version 4.0.20. Chart version 7.8.0 uses MongoDB version 4.0.14 and the next version 7.8.1 uses MongoDB version 4.2.2.

![image](https://user-images.githubusercontent.com/18659982/94130996-b3317d00-fe5d-11ea-82c8-9eddfcf7d07f.png)

Using a stateful set to deploy as a stand-alone instance has been introduced in version 7.8.0 but it has a bug and it has been fixed in version 7.8.7 but it uses MongoDB version 4.2.3.

Reply:
Yes, I thought that it might be good to use the same version in the chart that we also use in the unit and integration tests, wouldn't you agree? However, this doesn't necessarily mean that we must use 4.0.20 in the chart. We might as well upgrade to a more recent version (4.2.x) in the unit/integration tests. WDYT?

Reply:
I agree that it is good to use the same version. Yes, it is also better that we upgrade to a more recent version in our unit/integration tests.